ui/app/rif/pages/rns/renew/index.js

```diff
@@ -1,12 +1,31 @@
 import React, {Component} from 'react'
 import {connect} from 'react-redux'
+import DepositOnChannel from '../../../components/lumino/deposit-on-channel';
+import OpenChannel from '../../../components/lumino/open-channel';
+import CloseChannel from '../../../components/lumino/close-channel';
 
 class Renew extends Component {
 
   static propTypes = {}
 
   render () {
-    return (<div className="body">Renew</div>);
+    return (<div className="body">
```

Comment:
I think this was used for testing and should not be here

Reply:
Yes, you are right, sorry about that.